config/initializers/secrets.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# Be sure to restart your server after modifying this file.
+
+require "pathname"
```

Comment:
If you then follow @ereslibre's suggestion, then remove this since it won't be needed anymore 😉